source/js/third-party/tags/wavedrom.js

```diff
@@ -0,0 +1,11 @@
+/* global NexT, CONFIG, WaveDrom */
+
+document.addEventListener('page:loaded', () => {
+  NexT.utils.getScript(CONFIG.wavedrom.js, {
+    condition: window.WaveDrom
+  }).then(() => {
+    NexT.utils.getScript(CONFIG.wavedrom_skin.js).then(() => {
```

Comment:
Condition for `CONFIG.wavedrom_skin.js` is `window.WaveSkin`

Reply:
Got it. Didn't add it since it is in the then block. It is fixed now.

Reply:
demo page is also updated now: https://r12f.com/posts/use-wavedrom-in-hexo/